packages/core/src/Badge/Badge.tsx

```diff
@@ -56,6 +56,9 @@ const sizes = {
     ${themeGet('textStyles.display0')}
     line-height: ${themeGet('lineHeights.standard')};
   `,
+  mediumCaption: css`
+    ${themeGet('typography.caption')}
+  `,
 }
```

Comment:
I'm not the biggest fan how this component labels `size` as the prop to manipulate text styling. I'm going with the name `mediumCaption` for now. If the test wins, I can update medium to use typography.caption so it may be the new default